supershaper.init

```diff
@@ -108,6 +108,14 @@ TC=tc
 # Where to store flow labels, accessible for other tools
 FLOWS_FILENAME="/run/tc-flows.csv"
 
+# Stab setting
+# Configure a stab to calculate a more accurate queue size and avoid wasting bandwidth.
+# PPPoE ADSL/VDSL over ATM
+STAB="stab linklayer ethernet overhead 40"
```

Comment:
@yutayu This number `40` is what I call a "magic number". You should include (in a comment) how it was calculated. Same for the number `10` below.